PROVESFlightControllerReference/Components/TcSecurityDeframer/Authenticator.cpp

```
@@ -97,10 +92,14 @@ PacketAuthenticator::KeyImportResult importHmacKey(const char* key, uint32_t& ke
     return {PacketAuthenticator::KeyImportStatus::Success, PSA_SUCCESS};
 }
 
+int32_t destroyHmacKey(uint32_t keyId) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)